### ISSUE TITLE
Implement user avatars

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#50]
### Who

- [ ] Anonymous visitor
- [x] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To upload their own user picture.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
